config/currency_iso.json

```diff
@@ -1584,6 +1584,7 @@
     "iso_code": "NIO",
     "name": "Nicaraguan Córdoba",
     "symbol": "C$",
+    "disambiguate_symbol": "NIO$",
```

Comment:
Ok, I see. I guess we can add this, since it's a very rare use case when you'll have these two currencies formatted with their symbols.